Classes/PhDThesisPSnPDF.cls

```diff
@@ -14,7 +14,7 @@
 \newcommand\fileversion{2.2.2}
 \newcommand\filedate{2016/10/20}
 \NeedsTeXFormat{LaTeX2e}
-\ProvidesClass{PhDThesisPSnPDF}[\filedate\space A PhD thesis class file
+\ProvidesClass{Classes/PhDThesisPSnPDF}[\filedate\space A PhD thesis class file
```

Comment:
agreed, didn't intend to provide an overall fix. 
This only removes the warning using windows, probably not even linux/osX (at least on my setup). 
My apologies for the pull request. @kks32 , could you rollback my pull requests?